pkg/config/config.go

```diff
@@ -266,7 +266,6 @@ func (c *Config) GetStepConfig(flagValues map[string]interface{}, paramJSON stri
 			return StepConfig{}, err
 		}
 		if vaultClient != nil {
-			defer vaultClient.MustRevokeToken()
```

Comment:
Vault token will be revoked in the end of each step. See PR: https://github.com/SAP/jenkins-library/pull/5131/files